include/deal.II/distributed/tria_base.h

```diff
@@ -201,6 +202,20 @@ namespace parallel
     const std::set<types::subdomain_id> &
     level_ghost_owners() const;
 
+    /**
+     * Return partitioner for the global indices of the cells on the active
+     * level of the triangulation.
+     */
+    const Utilities::MPI::Partitioner &
+    global_active_cell_index_partitioner() const;
+
+    /**
+     * Return partitioner for the global indices of the cells on the given @p
+     * level of the triangulation.
+     */
+    const Utilities::MPI::Partitioner &
+    global_level_cell_index_partitioner(const unsigned int level) const;
+
```

Comment:
A question on the place for this function: If we envision these functions to be often, I would prefer to have the partitioners in the base class `Triangulation` already, to avoid downcasting in many places. After all, a partitioner is also valid (albeit boring) on a serial triangulation.

Reply:
Wait, we also provide the MPI communicator only in this function, so we better stick to this class.